lib/prerequisite.js

```diff
@@ -106,18 +106,10 @@ module.exports = (input, pkg, options) => {
 					tagPrefix = await getTagVersionPrefix();
 				} catch (_) {}
 
-				try {
-					const {stdout: revInfo} = await execa.stdout('git', ['rev-parse', '--quiet', '--verify', `refs/tags/${tagPrefix}${newVersion}`]);
+				const tagExists = await doesTagExist(`${tagPrefix}${newVersion}`);
```

Comment:
Same as [my comment on `valid` and `validRemote()`](https://github.com/sindresorhus/np/pull/326/files#r247308063) (refs [this comment](https://github.com/sindresorhus/np/pull/326/files#r247308025)).